docker/dump-crontab

```diff
@@ -1,6 +1,14 @@
-# Full Data dumps creation for backup and public use, every Sunday
-00 00 * * 0 /code/listenbrainz/admin/create-dumps.sh full
+# Full dump creation takes the ID of the last incremental
+# dump and creates a full dump for that ID. So, we'll
+# start creating full dumps and incremental dumps in pairs
+# for now.
```

Comment:
And sometime later when things are running we do full dumps less often?

Reply:
yes! I'm planning on using a few weeks of ishaan's gsoc to get incremental dumps support.